src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -232,14 +235,63 @@ protected void onStartup() {
         super.onStartup();
 
         initializeReference();
+
         initializeReads();
+
         initializeFeatures();
+
         initializeIntervals(); // Must be initialized last, since intervals currently require a sequence dictionary from another data source
 
+        validateSequenceDictionaries();
+
         checkToolRequirements();
     }
 
     /**
+     * Validate all sequence dictionaries by checking them against each other.
+     */
+    private void validateSequenceDictionaries() {
+        SAMSequenceDictionary refDict = null;
+        if (hasReference()){
+            refDict = reference.getSequenceDictionary();
+        }
```

Comment:
Use ? : syntax here and below: 

`SAMSequenceDictionary refDict = hasReference() ? reference.getSequenceDictionary() : null`